frontends/web/src/locales/en/app.json

```diff
@@ -64,12 +64,13 @@
   },
   "aopp": {
     "addressRequest": "{{host}} is requesting a receiving address.",
+    "addressRequestWithLogo": "is requesting a receiving address",
```

Comment:
looks like this and the above could be unified into one entry

Reply:
on 2nd thought, maybe it's better to formulate this one without assuming that the logo is in front. it is possible it can't be translated this way in some languages due to different grammar.

above, there is {{host}}, so in different languages it can be in a different position, but not here.

Reply:
Maybe: `There is an address request by:` in front of the logo?

then the same text can be used with the domain too - one entry less to translate, and it works for both cases.

Reply:
I'll defer to @jadzeidan and @Stadicus